src/Raven.Client/ServerWide/Commands/GetDatabaseTopologyCommand.cs

```diff
@@ -17,10 +18,11 @@ public GetDatabaseTopologyCommand()
             Timeout = TimeSpan.FromSeconds(15);
         }
 
-        public GetDatabaseTopologyCommand(string debugTag, Guid? applicationIdentifier)
+        public GetDatabaseTopologyCommand(string debugTag, Guid? applicationIdentifier, bool usePrivateUrls = false)
```

Comment:
create a separate internal ctor for this for now

Reply:
done